OBAApplicationDelegate.m

```diff
@@ -63,7 +68,14 @@ - (id)init {
                                                                                 [self writeSetRegionAutomatically:YES];
                                                                                 [self.regionHelper updateNearestRegion];
                                                                             }];
-
+        self.recentStopsObserver = [[NSNotificationCenter defaultCenter] addObserverForName:OBAMostRecentStopsChangedNotification
```

Comment:
I added a dealloc to remove observer recentStopsObserver and the regionObserver above it. The dealloc will never be called on the OBAApplicationDelegate, but I agree it's good form to remove the observer.